tests/testData/gen3TestRepo/gen3.sqlite3

```diff
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:ad3b14da4a8d0453a59f0355ce12244e5f99476f939e5b9614199a47c008d46f
+oid sha256:184a4b78d097bbef867c844b2fd9814619a9928be53b13d330152a29a4f6e3a1
```

Comment:
Did you actually change any data in the test repository? If not please don't update the repo.